### ISSUE TITLE
{ /res auto } Is not working as expected.

### ISSUE THREAD

**Issue by klaidoshka** [Zrips/Residence#786]
Hello, it's about `/res auto` and `groups.yml` 👋 (And sorry if my mentions of settings aren't right, I didn't open groups.yml for some long time)

**The calculations of the `/res auto` command (I suppose?) aren't working as wanted.**

The thing is that it is impossible to create 50x50 residence when a group has set MinEast to >0 and MaxEast to 50 (for example), it will always say that 'you' can't create 50x50 residence, it's above your limits, even though you will be able to create it via `/res create`... It's the same for any other Min[Side] probably too. However, if you lower the MinEast to 0 or set MaxEast to 53, `/res auto` will work for 50x50.

In short, the `/res auto` takes the range between Min & Max, not the Max value, which is not right, I guess.
